internal/rps/message.go

```diff
@@ -128,6 +128,7 @@ func (p Payload) CreateMessageRequest(flags rpc.Flags) (RPSMessage, error) {
 	if err != nil {
 		return message, err
 	}
+	// Update with AMT password for activated devices
 	if flags.Password == "" && payload.CurrentMode != 0 {
```

Comment:
since these condition both look at CurrentMode != 0, perhaps make that the outer condition, and if/else inside of it looking at flags.password